### PR TITLE
Added manual way to ignore task

### PR DESCRIPTION
**Pull request by matllubos**

### CODE REVIEW

File: django_celery_extensions/task.py [hunk 1]
```diff
@@ -33,9 +33,6 @@
 logger = logging.getLogger(__name__)
 
 
-cache = caches[settings.CACHE_NAME]
```
Comment (matllubos) [inline django_celery_extensions/task.py:36]: I want to override cache settings in tests

File: django_celery_extensions/task.py [hunk 2]
```diff
@@ -644,6 +651,9 @@ def get_django_command_task(command_name):
 
 
 def auto_convert_commands_to_tasks():
+    from django.core.management.base import BaseCommand
+    BaseCommand.stealth_options = tuple(BaseCommand.stealth_options) + ('celery_task',)
```
Comment (matllubos) [inline django_celery_extensions/task.py:655]: this is little hack I need somehow send celery task into command. This is only way how to do it

File: django_celery_extensions/task.py [hunk 3]
```diff
@@ -465,11 +466,11 @@ def _trigger(self, result, args, kwargs, invocation_id, task_id=None, eta=None,
 
         ignore_task_after_success = (
             ignore_task_after_success if ignore_task_after_success is not None
-            else self.ignore_task_after_success_timedelta is not None
+            else self.ignore_task_timedelta is not None
```
Comment (Formulka) [inline django_celery_extensions/task.py:469]: Shouldn't this be `self.ignore_task_after_success`?

Comment (matllubos) [reply django_celery_extensions/task.py:469]: No this must work for manual and automatic (after success) ignore

**Review by rubickcz — APPROVED** [review]

**Pull request merged**